src/seedu/addressbook/AddressBook.java

```diff
@@ -486,12 +486,38 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
+            
+            convertToUppCase(keywords);
+            convertToUppCase(wordsInName);
+            
             if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
+    
+    
+    /**
+     * Convert names to upper case for comparison later in getPersonsWithNameContainingAnyKeyword
```

Comment:
the comments should be in active form 
a better version would be `Convert**s** names to ...`